rundetection/rules/sans_rules.py

```diff
@@ -121,57 +106,104 @@ def _set_transmission_file(job_request: JobRequest, sample_title: str, sans_file
-        logger.info("LOQ trans found %s", trans_run_number)
+        logger.info("%s trans found %s", job_request.instrument, trans_run_number)
     else:
         trans_run_number = str(job_request.run_number)
-        logger.info("LOQ trans set as scatter %s", trans_run_number)
+        logger.info("%s trans set as scatter %s", job_request.instrument, trans_run_number)
     if trans_run_number is not None:
         job_request.additional_values["scatter_transmission"] = trans_run_number
 
 
 def _set_can_files(can_title: str | None, job_request: JobRequest, sans_files: list[SansFileData]) -> None:
     if can_title is not None:
         can_scatter = _find_can_scatter_file(sans_files=sans_files, can_title=can_title)
-        logger.info("LOQ can scatter found %s", can_scatter)
+        logger.info("%s can scatter found %s", job_request.instrument, can_scatter)
         if can_scatter is not None:
             job_request.additional_values["can_scatter"] = can_scatter.run_number
 
         # If using M4 monitor then can scatter is the transmission
         if not job_request.additional_values["included_trans_as_scatter"]:
             can_trans = _find_can_trans_file(sans_files=sans_files, can_title=can_title)
-            logger.info("LOQ can trans found %s", can_trans)
+            logger.info("%s can trans found %s", job_request.instrument, can_trans)
         else:
             can_trans = can_scatter
-            logger.info("LOQ can trans set as scatter %s", can_scatter)
+            logger.info("%s can trans set as scatter %s", job_request.instrument, can_scatter)
         if can_trans is not None and can_scatter is not None:
             job_request.additional_values["can_transmission"] = can_trans.run_number
 
 
 def _set_direct_files(job_request: JobRequest, sans_files: list[SansFileData]) -> None:
     direct_file = _find_direct_file(sans_files=sans_files)
-    logger.info("LOQ direct files found %s", direct_file)
+    logger.info("%s direct files found %s", job_request.instrument, direct_file)
     if direct_file is not None:
         if "scatter_transmission" in job_request.additional_values:
             job_request.additional_values["scatter_direct"] = direct_file.run_number
         if "can_scatter" in job_request.additional_values and "can_transmission" in job_request.additional_values:
             job_request.additional_values["can_direct"] = direct_file.run_number
 
 
-class LoqFindFiles(Rule[bool]):
+class CheckIfScatterSANS(Rule[bool]):
+    def __init__(self, value: bool):
+        super().__init__(value)
+        self.should_be_first = True
+
+    def verify(self, job_request: JobRequest) -> None:
+        if not job_request.experiment_title.endswith("_SANS/TRANS") and not job_request.experiment_title.endswith(
+            "_SANS"
+        ):
+            job_request.will_reduce = False
+            logger.error("Not a scatter run. Does not have _SANS or _SANS/TRANS at the end of the experiment title.")
```

Comment:
Can we drop these from error logs, since they aren't really errors at all and expected behaviours. Same below as well